.github/workflows/release.yml

```
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  Publish:
+    runs-on: ubuntu-latest
+    environment: release
+    permissions:
+      id-token: write
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.10"
+
+    - name: Install Nox
+      run: |
+        python -m pip install nox
+
+    - name: Build a binary wheel and a source tarball
+      run: |
+        nox -e build
+
+    - name: Publish package distributions to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        print-hash: true
+        skip-existing: true
```

```suggestion
        skip-existing: false
```

This should likely only be used when it comes to TestPyPI rather than prod, since test would result in duplicates that can happen but shouldn't be the case with production releases.

---

Need to update your doc for suggested release.yml, since that was cut & paste from it
Will fix tomorrow along with the others

---

done